test/integration/main.bats

```diff
@@ -3,7 +3,7 @@
 load helpers
 
 @test "test that create doesn't accept any arugments" {
-        run swarm create derpderpderp
```

Comment:
Most of whitespace used so far are tabs, but this line somehow contained spaces. My editor just re-formatted it to all tabs.